dsa-23au/java-dsa/weiss-pham-dmv/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>dev.codewithfriends</groupId>
+  <artifactId>weiss-pham-dmv</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>weiss-pham-dmv</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
```

Comment:
[Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)

When are we supposed to finish the DMV project?